services/api/routes/authenticated.js

```diff
@@ -190,7 +192,8 @@ var routes = [
                     Joi.number().integer().required(),
                     Joi.string().regex(/^\$(\d+)\.(.*)$/)
                   ),
-                  title: Joi.string().max(256).required()
+                  title: Joi.string().max(256).required(),
```

Comment:
(moved from below, derp)

totally is, but I was told to set desc to a max of 100 :P